mlflow/tracking/client.py

```diff
@@ -2738,15 +2749,24 @@ def print_model_version_info(mv):
             Version: 1
             Tags: {'t': '1'}
         """
+        _validate_model_version_or_stage_exists(version, stage)
+        if stage:
+            latest_versions = self.get_latest_versions(name, stages=[stage])
+            version = latest_versions[0].version if latest_versions else None
```

Comment:
If `version` is `None` here, should we throw a clear exception?

Reply:
my opinion is validation within `_get_registry_client().set_model_version_tag` will handle and raise appropriate the error msg.  https://github.com/mlflow/mlflow/blob/4f5d3f095da9ed6699941d8aacb9925488ff2fb5/mlflow/store/model_registry/sqlalchemy_store.py#L813

Reply:
I think the error message might be confusing, since the user is passing a stage and isn't explicitly passing `version=None`. Can we raise a separate exception here?

Reply:
Got it !! Let me add it ! I also hope you are fine with this change https://github.com/mlflow/mlflow/pull/6185/files#diff-b998d62895a5fdf2439e985b290e6240b7d6bd7417d214f3ac59c763e592a6ddR20

Reply:
Thanks @subramaniam02 ! That change looks great :)